components/server/src/container-module.ts

```diff
@@ -108,6 +110,7 @@ export const productionContainerModule = new ContainerModule((bind, unbind, isBo
     bind(WebsocketConnectionManager).toDynamicValue(ctx =>
         new WebsocketConnectionManager<GitpodClient, GitpodServer>(() => ctx.container.get<GitpodServerImpl<GitpodClient, GitpodServer>>(GitpodServerImpl))
     ).inSingletonScope();
+    bind(ResourceAccessGuard).to(OwnerResourceGuard).inSingletonScope();
```

Comment:
Is this binding ever used? It does not seem that `OwnerResourceGuard` can be instantiated without a user?